packages/opentelemetry-exporter-collector/src/util.ts

```diff
@@ -33,3 +33,9 @@ export function parseHeaders(
   });
   return headers;
 }
+
+export function appendResourcePathToUrlIfNotPresent(url: string, path: string): string {
```

Comment:
i guess you could export this util since proto exporter have the base one as dependencies to avoid duplicating code but thats not a requirement for me

Reply:
@vmarchaud  I'm going to leave as is for now as the tests started failing locally after I made that change unless it is obvious and guidance is provided or someone else can add onto this PR and I can learn from it.

Reply:
you should be able to simply export only this one function, once you have it then you can easily use it in all other packages

Reply:
Branch with changes: https://github.com/longility/opentelemetry-js/commit/2148d7f50ae9ed55c7c5ac8c253f20ffe58e1489

I'm not getting typescript errors in vscode, and it was able to find the function naturally as expected. However, I'm getting this error after my changes when running the tests locally:
```
  1) CollectorMetricExporter - node with proto over http
       when configuring via environment
         should use url defined in env:
     TypeError: exporter_collector_1.appendResourcePathToUrlIfNotPresent is not a function
      at CollectorMetricExporter.getDefaultUrl (src/CollectorMetricExporter.ts:22:44)
      at new CollectorExporterBase (/Users/longility/Repos/opentelemetry-js/packages/opentelemetry-exporter-collector/src/CollectorExporterBase.ts:45:21)
      at new CollectorExporterNodeBase (/Users/longility/Repos/opentelemetry-js/packages/opentelemetry-exporter-collector/src/platform/node/CollectorExporterNodeBase.ts:45:5)
      at new CollectorExporterNodeBase (src/CollectorExporterNodeBase.ts:18:192)
      at new CollectorMetricExporter (src/CollectorMetricExporter.ts:18:197)
      at Context.<anonymous> (test/CollectorMetricExporter.test.ts:60:33)
      at processImmediate (internal/timers.js:464:21)
      ```

Any insight?

Reply:
you are not exporting this function from `@opentelemetry/exporter-collector`

Reply:
I thought I did.

![image](https://user-images.githubusercontent.com/720635/131383052-e8600ad7-3ec4-4cf7-8f39-e5600cb35068.png)

Unless you meant something else. Do I need to split the PR as there is a dependency? This is where I'm not as familiar and need more guidance.

Reply:
once you do it, make sure you rebuild package before using it, to see changes immediately just run "npm run watch"

Reply:
I also don't see the change in the github UI. can you provide a link instead of just a screenshot?

Reply:
Sorry, I wasn't clear. I provided a link to the wip branch and not part of this PR. Here it is again: https://github.com/longility/opentelemetry-js/commit/2148d7f50ae9ed55c7c5ac8c253f20ffe58e1489

I will try what you suggested @obecny and if it passes, I will put on this PR.

Reply:
`npm run compile` in exporter collector did the trick. That was my missing piece. Thanks!

I updated the PR.